### PR TITLE
Step 7: Creating Contact Me Section

### PR DESCRIPTION
**Pull request by leehaney254**
## Step 7: Creating Contact Me Section
In this step I:
- Added contact Me section
- Linked my form to formspree
- updated my readme.md file

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -358,11 +366,75 @@ ul {
...
+
+#form-ul {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 24px;
+  width: 327px;
+  height: 330px;
+  list-style-type: none;
+}
```
Comment (jerryowusu) [inline styles.css:417]: Kindly stick to the design as much as possible. There are some disparities between your design and that of the figma especially with regards to the spacing and positioning. There is a blank space to the right side which increases with increasing screen width as shown below. :1234: Kindly check and fix that.

Your design            |  The design template
:-------------------------:|:-------------------------:
Screenshot from 2022-11-01 15-59-00  |  Screenshot from 2022-11-01 15-58-35 


Your design            |  The design template
:-------------------------:|:-------------------------:
Screenshot from 2022-11-01 15-46-31  |  Screenshot from 2022-11-01 15-46-42

File: styles.css [hunk 2]
```diff
@@ -358,11 +366,75 @@ ul {
...
+textarea {
+  width: 327px;
+  height: 114px;
+  border: none;
+  background: #e5e5e5;
+}
+
 button:hover {
   background: #fff5e1;
   border: 1px solid #f55800;
```
Comment (jerryowusu) [inline styles.css:455]: The button interactions such as hover, active enable etc is not working for the form button, kindly check and fix that. :pray:

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @Team

Good job so far!
Your project is looking fantastic :star_struck: 

However, I've found some room for improvement that can help you accomplish a more readable/clean code and make your project look professional :sunglasses:

so-close


## Highlights
All linters are passing :heavy_check_mark: 
Good commit messages :heavy_check_mark: 
Excellent pull request title and descriptive summary :1st_place_medal: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [x] Your readme documentation lacks a good description and a license reference. Remember, a good documentation not only make your project looks professional, it is also a testament as to how discipline and committed you are to your work. :hammer_and_wrench:  Kindly check and fix that accordingly.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by leehaney254** [comment]
Hello!
- I have updated my readme file. :smile: 
- Hover is working on the form button. :100: 
- There are no white spaces growing. :+1:

**Review by Olamarx — APPROVED** [review]
Hi @leehaney254, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linters passed ✔️ 
- Descriptive commit messages 👍🏼 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -6,12 +6,14 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="preconnect" href="https://fonts.googleapis.com" />
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link rel="preconnect" href="https://fonts.googleapis.com" />
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
     <link
       href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto&display=swap"
       rel="stylesheet"
     />
```
Comment (Talha-Hanif5666) [inline index.html:12]: I think it is better to close the link tag in the same line on which you ended the text.  Kindly, please move "/>" to the previous line i-e line#11.  By doing this, your code will look more professional.

**Review by Talha-Hanif5666 — COMMENTED** [review]
**Hi @Talha-Hanif5666,**
157397398-00904e26-60ec-46cd-9510-240661c250ef

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



**Highlights**
 No linters errors :heavy_check_mark:
 Descriptive PR :heavy_check_mark:
Good commit messages :heavy_check_mark:

**Changes Required** :recycle: 
 Please kindly check the comments under the review point_down point_down
**Optional suggestions**
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

**Comment by Rana-Mudassir** [comment]
Everything is perfect. Linters are checked. The formatting is good. The code is clean to go.
Cheers and Happy coding!👏👏👏

**Review by Lucas-Erkana — COMMENTED** [review]
Hi @leehaney254 
our project is complete! There is nothing else to say other than... Great job 👏
Congratulations! 🎉

TO HIGHLIGHT:

Great job Implementing the changes provided by the previous reviewers ✅
Your project is professional ✅
Your repo is professional ✅
Well structured files ✅
No Linter errors ✅
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f686c516559335062684833692f67697068792e676966

[OPTIONAL] I suggest adding the MIT.md file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉
he MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯
very comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!👏👏👏

**Pull request merged**